src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -348,6 +348,7 @@ GlobalSettingsPage.title             = Global Settings
 GlobalSettingsPage.verboseExceptions = Include stack trace in service exception
 GlobalSettingsPage.verboseMessaging  = Verbose XML output (pretty print) 
 GlobalSettingsPage.xmlExternalEntitiesEnabled = Unrestricted XML External Entity Resolution (security risk)
+GlobalSettingsPage.trailingSlashMatch = Match URLs with trailing slash (Requires restart)
```

Comment:
Nice that you added it to the label, as it's unusual. The screenshot is missing that bit though.

Reply:
screenshot updated